service/src/main/java/uk/gov/hmcts/reform/fpl/NotifyTemplates.java

```diff
@@ -56,4 +56,7 @@ private NotifyTemplates() {
     public static final String FURTHER_EVIDENCE_UPLOADED_NOTIFICATION_TEMPLATE = "917c07b2-b414-41ab-9d0b-14a570fad413";
     public static final String REGISTERED_RESPONDENT_SOLICITOR_TEMPLATE = "c336fa70-497c-4f45-8cc9-2f2dc16c3ca9";
     public static final String UNREGISTERED_RESPONDENT_SOLICITOR_TEMPLATE = "6307a132-8944-4b99-9b4f-5af0cd12f1e8";
+    public static final String UNREGISTERED_RESPONDENT_SOLICICTOR = "6307a132-8944-4b99-9b4f-5af0cd12f1e8";
```

Comment:
looks that UNREGISTERED_RESPONDENT_SOLICICTOR is duplicate of UNREGISTERED_RESPONDENT_SOLICITOR_TEMPLATE and is not used